shadows/framework/src/main/java/org/robolectric/shadows/util/SQLiteLibraryLoader.java

```diff
@@ -77,6 +81,13 @@ public boolean isLoaded() {
     return loaded;
   }
 
+  public static boolean isOsSupported() {
```

Comment:
@hoisie could you accept adding this public API?

Reply:
I will merge it firstly, and we can improve it at later commits before next release.